app/_data/tables/install_options_34x.yml

```diff
@@ -1,84 +1,75 @@
 columns:
-  - name: Kong OSS
+  - name: Platform
+  - name: Open Source
+    subtitle: An open-source package containing the basic API gateway functionality and open-source plugins.
     key: oss
   - name: Kong Gateway Enterprise
+    subtitle: Kong’s API gateway with added functionality.
     key: enterprise
-  - name: Officially Supported
-    key: support
 features:
-  - name: "Packages"
+  - name: "Docker"
     items:
-      - name: "Debian"
-        url: /gateway/VERSION/install/linux/debian/
+      - name: "Debian (bullseye-slim)"
         oss: true
         enterprise: true
         support: true
+        url: /gateway/VERSION/install/docker/build-custom-images/
         icon: /assets/images/icons/third-party/debian-logo.jpg
-      - name: "Ubuntu"
-        url: /gateway/VERSION/install/linux/ubuntu/
+      - name: "RHEL (8-ubi)"
         oss: true
         enterprise: true
         support: true
-        icon: /assets/images/icons/third-party/ubuntu.png
-      - name: "RedHat"
-        url: /gateway/VERSION/install/linux/rhel/
+        url: /gateway/VERSION/install/docker/build-custom-images/
+        icon: /assets/images/icons/third-party/rhel.jpg
+      - name: "Ubuntu"
         oss: true
         enterprise: true
         support: true
-        icon: /assets/images/icons/third-party/rhel.jpg
-      - name: "Amazon Linux 2"
-        url: /gateway/VERSION/install/linux/amazon-linux/
+        url: /gateway/VERSION/install/docker/
+        icon: /assets/images/icons/third-party/ubuntu.png
+      - name: "Amazon Linux (2 and 2023)"
         oss: true
         enterprise: true
         support: true
+        url: https://hub.docker.com/r/kong/kong-gateway/tags?page=1&name=amazonlinux
         icon: /assets/images/icons/third-party/amazon-linux.png
   - name: "Kubernetes"
```

Comment:
The Kubernetes, KIC, and Docker OSS installs on `/gateway/VERSION/install/docker/build-custom-images/` all get the `tab=oss` generated in their URLs, but those tabs don't exist on those pages. It doesn't really cause any problems, it just doesn't do anything.

Reply:
🤦 I should have caught this, nice catch!